sky/templates/kubernetes-sshjump.yml.j2

```diff
@@ -0,0 +1,83 @@
+pod_spec:
+    apiVersion: v1
+    kind: Pod
+    metadata:
+        name: {{ name }}
+        labels:
+            component: {{ name }}
+    spec:
+        serviceAccountName: sshjump
+        volumes:
+          - name: secret-volume
+            secret:
+                secretName: {{ secret }}
+        containers:
+          - name: {{ name }}
+            imagePullPolicy: IfNotPresent
+            image: {{ image }}
+            command: ["python3", "-u", "/skypilot/sky/utils/kubernetes/sshjump-lcm.py"]

```

Comment:
Can we add a breif comment on how this script ran from the container will remove jump pod itself and the service with the script when there isn't any k8s instance for certain time?

Reply:
Yes, I am adding this as a comment in `utils/kubernetes/sshjump-lcm.py` (on top).